File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    if 500000 <= price :
        return int(price * 0.8)
        
    elif 300000 <= price and price < 500000 :
        return int(price * 0.9)
        
    elif 100000 <= price and price < 300000 :
        return int(price * 0.95)
        
    else:
        return price;
        

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.cs
using System;

class Solution {
    public int solution(int price) {
        if( 500000 <= price ){
            price = (int)(price * 0.8);
        }
        else if( 300000 <= price && price < 500000 ){
            price = (int)(price * 0.9);
        }
        else if( 100000 <= price && price < 300000 ){
            price = (int)(price * 0.95);
        }
        else{
            price = price;
        }
            
        int answer = price;
        return answer;
    }
}